openslides_backend/action/actions/user/base_import.py

```diff
@@ -21,13 +21,15 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
 
         if self.import_state != ImportState.ERROR:
             rows = self.flatten_copied_object_fields(
-                self.handle_remove_and_group_fields
+                self.handle_remove_email_and_group_fields
             )
             self.create_other_actions(rows)
 
         return {}
 
-    def handle_remove_and_group_fields(self, entry: Dict[str, Any]) -> Dict[str, Any]:
+    def handle_remove_email_and_group_fields(
```

Comment:
Don't change the method name. "email" is removed/poped like gender and other fields